lua/wire/client/text_editor/texteditor.lua

```diff
@@ -1152,26 +1119,23 @@ function EDITOR:CountFinds( str )
 	end
 
 	if whole_word_only then
-		local pattern = "([^a-zA-Z0-9_])"..str.."([^a-zA-Z0-9_])"
-		txt = " " .. txt
-		local num1, num2 = 0, 0
-		txt, num1 = txt:gsub( pattern, "%1%2" )
-		if txt == "" then return num1 end
-		txt, num2 = txt:gsub( pattern, "%1%2" )
-		return num1+num2
-	else
-		local num
-		txt, num = txt:gsub( str, "" )
+		str = "%f[%w_]()" .. str .. "%f[^%w_]()"
+	end
+
+	local _, num = txt:gsub(str, "")
 		return num
```

Comment:
This and the `end` after it should be tabbed back. Also, right above this there's a merge conflict.